task/01-strings-tasks.js

```diff
@@ -221,7 +224,9 @@ function getRectangleString(width, height) {
  *
  */
 function encodeToRot13(str) {
-    throw new Error('Not implemented');
+    const alpha = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+    const flipped = 'NOPQRSTUVWXYZABCDEFGHIJKLMnopqrstuvwxyzabcdefghijklm';
```

Comment:
Despite the fact that the tests are being passed, this will not be the best solution.

`Hint`: use `charCodeAt` method